sdcm/provision/scylla_yaml/scylla_yaml.py

```diff
@@ -0,0 +1,343 @@
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation; either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright (c) 2021 ScyllaDB
+
+
+from typing import List, Literal, Union
+
+from pydantic import Field, validator
+
+from sdcm.provision.common.utils import BaseModel
+from sdcm.provision.scylla_yaml.auxiliaries import RequestSchedulerOptions, EndPointSnitchType, SeedProvider, \
+    ServerEncryptionOptions, ClientEncryptionOptions
+
+
+class ScyllaYaml(BaseModel):  # pylint: disable=too-few-public-methods
```

Comment:
I propose to rename it. "yaml" is only the filetype we use to store what we really have. And it is "scylla cluster configuration" in fact. So, name must be more closer to the real idea under it.

Reply:
Currently, Scylla uses `scylla.yaml` and this is how all people know it. If it will be changed in the future (which I doubt) we will rename it.